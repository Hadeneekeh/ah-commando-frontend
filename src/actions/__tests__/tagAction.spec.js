// import configureMockStore from 'redux-mock-store';

// import thunk from 'redux-thunk';
// import nock from 'nock';
// import { getArticlesWithTagFromDb } from '@Actions/tagAction';

// const mockStore = configureMockStore();
