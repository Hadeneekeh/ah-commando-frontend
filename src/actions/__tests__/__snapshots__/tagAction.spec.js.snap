// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag action tag action test tagAction should return expected value 1`] = `[Function]`;

exports[`tag action test view articles with tag action should display articles with tag 1`] = `
Array [
  Object {
    "payload": Array [],
    "tag": Object {},
    "type": "GET_ARTICLES_WITH_TAG",
  },
]
`;
